Mathlib/Tactic/Linter/DeprecatedSyntaxLinter.lean

```diff
@@ -112,6 +147,17 @@ def getDeprecatedSyntax : Syntax → Array (SyntaxNodeKind × Syntax × MessageD
       rargs.push (kind, stx,
         "The `admit` tactic is discouraged: \
          please strongly consider using the synonymous `sorry` instead.")
+    | ``Lean.Parser.Tactic.decide =>
+      if isDecideNative stx then
+        rargs.push (kind, stx, "Using `decide +native` is not allowed in mathlib:\n\
+        because it trusts the entire Lean compiler (not just the Lean kernel),\n\
```

Comment:
```suggestion
        rargs.push (kind, stx, "Using `decide +native` is not allowed in mathlib: \
        because it trusts the entire Lean compiler (not just the Lean kernel), \
```
I think that I prefer if line breaks are handled by the infoview formatter, since it is aware of the actual space that is available to display the messages.

Similarly below.